### PR TITLE
Firestore Experiment

### LINKED ISSUE #38: Develop experiment code for Firestore
Write some simple database and backend logic to test if Firestore can work on deployment.

### PR DESCRIPTION
**Pull request by malcolmrjones**
This is an experiment to check to see if Firestore will work on deployment. I was able to test Firestore on my local development environment using a workaround that uses Application Default Credentials. The route for the experiment is `/data` and a GET requests create 3 new movie documents and adds it to the movies collection. Then the movies data is retrieved from the database and returned. As of now and for testing purposes. I manually delete the movies collection when it gets too large.

**What the data looks like in the GCP Dashboard**
Screenshot 2020-07-06 at 14 29 44

**What the response looks like when running the first time**
Screenshot 2020-07-06 at 14 29 55

Fixes #38

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -7,6 +7,7 @@ var http = require('http');
 var path = require('path');
 var handlebars = require('express-handlebars')
 
+
```
Comment (danielabreo) [inline app.js:10]:
```suggestion
```

File: routes/data.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+async function readMovies() {
+    let movieList = [];
+    const moviesRef = firestore.collection('movies');
+    const snapshot = await moviesRef.get();
+    snapshot.forEach((movie) => {
+        movieList.push(movie.data());
+    });
+    return movieList;
+}
+
```
Comment (ccyang314) [inline routes/data.js:47]: nit: add JsDoc similar to `readMovies `.

File: routes/data.js [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+async function deleteAllMovies() {
+    let movieList = [];
```
Comment (ccyang314) [inline routes/data.js:49]: remove this unused variable.

File: routes/data.js [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+
+exports.view = function(req, res) {
+    createMovies();
+    const movies = readMovies();
+    movies.then((data) => {
+        res.send(data);
+    })
+};
+
+async function createMovies() {
```
Comment (ccyang314) [inline routes/data.js:14]: nit: add JsDoc similar to `readMovies`.

**Pull request merged**